shell/initialize/App.js

```diff
@@ -21,25 +21,10 @@ export default {
       key:      this.layoutName
     }, [layoutEl]);
 
-    const transitionEl = h('transition', {
-      props: {
-        name: 'layout',
-        mode: 'out-in'
-      },
-      on: {
-        beforeEnter(el) {
-          // Ensure to trigger scroll event after calling scrollBehavior
-          window.$nuxt.$nextTick(() => {
-            window.$nuxt.$emit('triggerScroll');
```

Comment:
I think you'll still need to emit this event on page transition.

It looks like there's some transitions which won't scroll to the top of the page when switching pages.


https://github.com/rancher/dashboard/assets/55104481/3c034cb9-f914-46f8-bdff-5f209eb52af7

Reply:
Well, this one was driving me crazy but I just realized that it never worked! I am running a 2.8.0-rc3 instance and the UI that is built with it fails on the move from Pods to Workloads (or vice versa... one of these two still has the problem).
I haven't been able to fix it but I think it's because the Pods list view is essentially the same as Workloads, although interestingly it does trigger a route change and everything seems to be correct through my investigation in https://github.com/rancher/dashboard/blob/master/shell/utils/router.scrollBehavior.js

Reply:
Fixed it! 💦  it wasn't registered as subtype of workloads... man, this one was tough

Reply:
Nice, find. Well done.